torch/_tensor_docs.py

```diff
@@ -2645,11 +2645,26 @@ def callable(a, b) -> number
 
 add_docstr_all('numpy',
                r"""
-numpy() -> numpy.ndarray
+numpy(force=False) -> numpy.ndarray
 
-Returns :attr:`self` tensor as a NumPy :class:`ndarray`. This tensor and the
+Returns :attr:`self` tensor as a NumPy :class:`ndarray`.
+
+If :attr:`force` is ``False``, tensor :attr:`self` and the
 returned :class:`ndarray` share the same underlying storage. Changes to
 :attr:`self` tensor will be reflected in the :class:`ndarray` and vice versa.
+If tensors storage is not CPU, or if :attr:`requires_grad` is ``True`` and
+:attr:`GradMode` is ``True``, or tensor is sprase, or :attr:`is_conj()`
+is ``True``, it throws exception.
+
+If :attr:`force` is ``True`` it tries to resolve issues, that would throw exception.
+It calls :func:`torch.Tensor.detach`, :func:`torch.Tensor.cpu` and if :attr:`is_conj()`
+is ``True`` it calls :func:`torch.Tensor.resolve_conj`.
```

Comment:
We always call resolve_conj right?

Reply:
Yes, just realized it's a no-op as well if is_conj() == False. Will fix that in docs as well as in the code.